components/Proposals/ProposalList.tsx

```diff
@@ -48,7 +49,9 @@ export default function ProposalList() {
               </Accordion.Control>
               <Accordion.Panel p="0" style={{ padding: '0' }}>
                 <Group justify="center" p="0">
-                  <MarketCard proposal={proposal} />
+                  <ProposalProvider fromProposal={proposal}>
```

Comment:
I think data should be organized in a way that does not require loading too much details yet. Using multiple context will make a ton of requests for data that we are not even sure we want to render (e.g. only display market prices, not possible action).

Keeping some market info in the AutocratContext enables just that.